model_metaswap/java/test/org/openda/model_metaswap/EsriAsciiGridSeriesDataObjectTest.java

```diff
@@ -1,21 +1,69 @@
 package org.openda.model_metaswap;
 
 import junit.framework.TestCase;
+import org.openda.exchange.dataobjects.EsriAsciiGridDataObject;
+import org.openda.exchange.dataobjects.EsriAsciiGridDataObjectTest;
+import org.openda.interfaces.IExchangeItem;
+import org.openda.interfaces.ITimeInfo;
+import org.openda.utils.OpenDaTestSupport;
+
+import java.io.File;
+import java.io.IOException;
 
 /**
  * Created by hummel on 2018-02-12.
  */
 public class EsriAsciiGridSeriesDataObjectTest extends TestCase {
-	public void testInitialize() throws Exception {
-	}
 
-	public void testGetExchangeItemIDs() throws Exception {
-	}
+	private OpenDaTestSupport testData = null;
+	File testRunDataDir = null;
 
-	public void testGetDataObjectExchangeItem() throws Exception {
+	protected void setUp() throws IOException {
+		testData = new OpenDaTestSupport(EsriAsciiGridSeriesDataObjectTest.class, "public", "model_metaswap");
+		testRunDataDir = new File(testData.getTestRunDataDir(), "gridSeries");
 	}
 
-	public void testFinish() throws Exception {
+	public void testParamAFiles() {
+		String[] arguments = {
+			"./meteo/paramA",
+			"outputFilePrefix=./paramA",
+			"timeStampFormat=yyyyMMdd",
+			"currentTime=20020101",
+			"targetTime=20020104"
+		};
+		EsriAsciiGridSeriesDataObject dataObject = new EsriAsciiGridSeriesDataObject();
+		dataObject.initialize(testRunDataDir, arguments);
+		IExchangeItem exchangeItem = dataObject.getDataObjectExchangeItem("paramA");
+		ITimeInfo timeInfo = exchangeItem.getTimeInfo();
+		assertNotNull(timeInfo);
+		double[] times = timeInfo.getTimes();
+		assertEquals(4, times.length);
+		assertEquals(52275d, times[0]);
+		assertEquals(52276d, times[1]);
+		assertEquals(52277d, times[2]);
+		assertEquals(52278d, times[3]);
+		double[] valuesAsDoubles = exchangeItem.getValuesAsDoubles();
+		assertEquals(160, valuesAsDoubles.length);
```

Comment:
There are a lot of double comparisons without some tolerance. I'd suggest to add an epsilon to all these asserts.